exploration-module/src/main/java/io/edpn/backend/exploration/adapter/persistence/MybatisSystemCoordinateRequestRepository.java

```diff
@@ -0,0 +1,29 @@
+package io.edpn.backend.exploration.adapter.persistence;
+
+import io.edpn.backend.exploration.adapter.persistence.entity.MybatisSystemCoordinateRequestEntity;
+import org.apache.ibatis.annotations.Delete;
+import org.apache.ibatis.annotations.Insert;
+import org.apache.ibatis.annotations.Result;
+import org.apache.ibatis.annotations.Results;
+import org.apache.ibatis.annotations.Select;
+
+import java.util.List;
+
+public interface MybatisSystemCoordinateRequestRepository {
+
+    @Insert("INSERT INTO system_coordinate_data_request (requesting_module, system_name) VALUES (#{requestingModule}, #{systemName})")
+    void insert(MybatisSystemCoordinateRequestEntity requestDataMessageEntity);
+
+    @Delete("DELETE FROM system_coordinate_data_request WHERE requesting_module = #{requestingModule} AND system_name = #{systemName}")
+    void delete(MybatisSystemCoordinateRequestEntity requestDataMessageEntity);
```

Comment:
Colour me curious, but what is the use-case for deleting a system?

Nefarious action by EDDN user, and some admin user has decided to nuke it from galactic orbit?